src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1041,102 +1041,184 @@ private bool IsValidFixedVariableInitializer(TypeSymbol declType, SourceLocalSym
             Debug.Assert(!ReferenceEquals(declType, null));
             Debug.Assert(declType.IsPointerType());
 
-            if (ReferenceEquals(initializerOpt, null))
+            if (initializerOpt?.HasAnyErrors != false)
             {
                 return false;
```

Comment:
Does a test cover this path?

Reply:
Added a test

---
In reply to: [167654777](https://github.com/dotnet/roslyn/pull/24469#discussion_r167654777) [](ancestors = 167654777)